client/components/Voting/voting.tsx

```diff
@@ -1,514 +1,26 @@
-import { Paginated } from '@feathersjs/feathers';
```

Comment:
This should be renamed to Voting.tsx to match with the Voting tag defined in pages/vote.tsx

Reply:
u right